systemtest/src/test/java/io/strimzi/systemtest/log/LoggingChangeST.java

```diff
@@ -1058,6 +1173,20 @@ void testDynamicallySetUnknownKafkaLoggerValue() {
     }
 
     @ParallelNamespaceTest
+    @TestDoc(
+        description = @Desc("Test dynamically setting Kafka external logging configurations to ensure logging updates without triggering unnecessary rolling updates, and verifies correct logging behavior."),
+        steps = {
+            @Step(value = "Create a ConfigMap with initial logging settings.", expected = "ConfigMap is created with the 'external-cm' name."),
+            @Step(value = "Apply external logging configuration to Kafka brokers.", expected = "Kafka brokers are configured with the specified external logging."),
+            @Step(value = "Verify no rolling update occurs after initial logging update.", expected = "No rolling update occurs and logging change is verified."),
+            @Step(value = "Update ConfigMap with new logger levels.", expected = "ConfigMap is updated and rolling update is expected."),
+            @Step(value = "Verify rolling update completion and new logging settings applied.", expected = "Rolling update is completed successfully and new settings are active.")
```

Comment:
I guess this is a bit different -> you deploy Kafka with some external CM configuration and then you change it, then you verify that there is no rolling update, right?